packages/taiko-client/prover/anchor_tx_validator/anchor_tx_validator.go

```diff
@@ -45,13 +47,15 @@ func (v *AnchorTxValidator) ValidateAnchorTx(tx *types.Transaction) error {
 		return fmt.Errorf("invalid TaikoL2.anchor transaction sender: %s", sender)
 	}
 
-	method, err := encoding.TaikoL2ABI.MethodById(tx.Data())
-	if err != nil {
-		return fmt.Errorf("failed to get TaikoL2.anchor transaction method: %w", err)
+	var method *abi.Method
+	if method, err = encoding.TaikoAnchorABI.MethodById(tx.Data()); err != nil {
+		if method, err = encoding.TaikoL2ABI.MethodById(tx.Data()); err != nil {
+			return fmt.Errorf("failed to get TaikoL2.anchor transaction method: %w", err)
```

Comment:
do we want to change all these TaikoL2 logs to `TaikoAnchor` ?

Reply:
Since for Ontake contracts, its name is still `TaikoL2`, will change all these logs after mainnnet has been in Pacaya fork. Then I will remove all Ontake code in client.